models/measurement.py

```diff
@@ -18,26 +16,22 @@
     aliased,
 )
 from sqlalchemy.ext.hybrid import hybrid_property
+import sqlalchemy as sql
 
 from models.orm_base import OrmBase
-from models.bioreplicate import Bioreplicate
-from models.strain import Strain
-from models.metabolite import Metabolite
+from models.measurement_technique import TECHNIQUE_NAMES
 from db import get_session
-from lib.db import execute_text
+from lib.conversion import convert_time
 
 
 class Measurement(OrmBase):
     __tablename__ = "Measurements"
 
-    class SubjectType(enum.Enum):
-        metabolite   = 'metabolite'
-        strain       = 'strain'
-        bioreplicate = 'bioreplicate'
-
```

Comment:
It's useful to list the different types that are available for a subject type, but unfortunately, this means that the value is always an "enum", so instead of accessing `measurement.subjectType`, we need to work with `measurement.subjectType.value`, and that's really easy to miss and annoying to debug.

It might be better to build a simple validation layer where we list the available values in a different way.